ur-packages/crypto-coin-identity/src/CoinIdentity.ts

```diff
@@ -116,13 +125,60 @@ export class CryptoCoinIdentity extends RegistryItem {
     const parts = url.split('://')[1].split('/')
     const subtypeParts = parts[0].split('.')
     if (subtypeParts.length > 1) {
-      const curve = subtypeParts[subtypeParts.length -1]
+      const curve = subtypeParts[subtypeParts.length - 1]
       const type = +parts[1]
-      const subTypes = subtypeParts.slice(0,subtypeParts.length -1)
+      const subTypes = subtypeParts.slice(0, subtypeParts.length - 1)
       return new CryptoCoinIdentity(curve as any, type, subTypes)
     }
     const curve = parts[0]
     const type = +parts[1]
     return new CryptoCoinIdentity(curve as any, type)
   }
+
+  /**
+   * Compare the current id to a given id
+   * @param coinIdentity CoinIdentity to compare the current one with
+   * @param comparison comparison method to check
+   * @returns boolean indicating if the comparison is valid
+   */
+  public compare = (coinIdentity: CryptoCoinIdentity, comparison: ComparisonMethod): boolean => {
+    const url = this.toURL().replace('bc-coin://', '')
+    const urlToCompare = coinIdentity.toURL().replace('bc-coin://', '')
+
+    return CryptoCoinIdentity.compareCoinIds(url, urlToCompare, comparison)
+  }
+
+  /**
+   * Compare a coin url, generated with '.toUrl()' method, with a different one
+   * @param coinUrl1 first coinIdentity as a url.
+   * @param coinUrl2 second coinIdentity as a url.
+   * @param comparison comparison method.
+   * @returns boolean indicating if the comparison is valid
+   */
+  static compareCoinIds(coinUrl1: string, coinUrl2: string, comparison: ComparisonMethod): boolean {
+    const dict = CryptoCoinIdentity.compareCoinIdsDict(coinUrl1, coinUrl2)
+    return dict[comparison]
+  }
+
+  /**
+   * Creates a dictionary for all comparison methods for two given coin urls, generated with '.toUrl()'.
+   * @param coinUrl1 first coinIdentity as a url.
+   * @param coinUrl2 second coinIdentity as a url.
+   * @returns dictionary indicating which comparison methods are true | false.
+   */
+  static compareCoinIdsDict(coinUrl1: string, coinUrl2: string) {
+    const isEqual = coinUrl1 === coinUrl2
+    const isChild = coinUrl1.startsWith(coinUrl2 + '/')
+    const isParent = coinUrl2.startsWith(coinUrl1 + '/')
```

Comment:
This comparison makes sense for our coin paths that we use in sync 1.3.

But when its coinID, then we wont have anything after the initial BIP44.

So these checks should check for the sub types :
`137.seckp256/60` <  `seckp256/60` = Polygon is sub type of Ethereum